source/plugins/go/src/go.mod

```diff
@@ -4,30 +4,366 @@ go 1.14
 
 require (
 	github.com/Azure/azure-kusto-go v0.5.2
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.12
+	github.com/Microsoft/go-winio v0.5.2
+	github.com/fluent/fluent-bit-go v0.0.0-20171103221316-c4a158a6e3a7
+	github.com/golang/mock v1.6.0
+	github.com/google/uuid v1.4.0
+	github.com/microsoft/ApplicationInsights-Go v0.4.4
+	github.com/tinylib/msgp v1.1.9
```

Comment:
1.  Does this update addresses the reported CVEs? 2. can you please do test for container log and prom metrics?

Reply:
I think so because this is not caught by trivy....Have tested the log and metrics scenario